mysql-test/include/wait_until_connected_again.inc

```diff
@@ -11,7 +11,7 @@ let $counter= 5000;
 let $mysql_errno= 9999;
 while ($mysql_errno)
 {
-  --error 0,ER_ACCESS_DENIED_ERROR,ER_SERVER_SHUTDOWN,ER_CONNECTION_KILLED,ER_LOCK_WAIT_TIMEOUT,2002,2006,2013,HA_ERR_NO_ENCRYPTION
+  --error 0,ER_ACCESS_DENIED_ERROR,ER_SERVER_SHUTDOWN,ER_CONNECTION_KILLED,ER_LOCK_WAIT_TIMEOUT,2002,2006,2013,2026,HA_ERR_NO_ENCRYPTION
```

Comment:
The source and meaning of these "client side" errors is very non-obvious in MTR tests, and hard to maintain or audit for completeness/correctness.

For example, when !2581 is merged, will _client-side_ error [2026 `CR_SSL_CONNECTION_ERROR`](https://github.com/mariadb-corporation/mariadb-connector-c/blob/master/include/errmsg.h#L44-L87) still be the right one to detect here? Or might it change to the _server-side_ `ER_SECURE_TRANSPORT_REQUIRED` which is added by that PR?

In general, seems to me like MTR ought to be aware of the symbolic names of [the client-side `CR_*` errors defined in `libmariadb`](https://github.com/mariadb-corporation/mariadb-connector-c/blob/master/include/errmsg.h#L44-L87), so that the tests don't contain these long lists of hard-to-discover and hard-to-remember integer constants.